packages/design-system/src/components/icon-button.tsx

```diff
@@ -47,7 +47,7 @@ export const IconButton = styled("button", {
         width: "$3",
         "&:hover, &:focus": {
           background: "none",
-          borderColor: "transparent",
+          border: "none",
```

Comment:
there was a problem with this: border is holding space, so if you remove border on hover, size of the icon will change and things may shift

Reply:
hmm, I can't find what was wrong ... maybe its something else